engine/include/Engine/Core/Rendering/Renderer/RendererGLFW_GL46.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ *	found in the top-level directory of this distribution.
+ */
+#pragma once
+
+#include "Engine/Core/Rendering/Renderer/IRenderer.hpp"
+
+namespace Engine::Core::Renderering
+{
+class RendererGLFW_GL46 : public IRenderer
```

Comment:
This is more of a "tip" than a commentary: you can also define a class `final` to prevent it from being derived again, and enable some optimization at compile-time, just like `final` for methods. For instanc,e, that would be correct:
```C++
class RendererGLFW_GL46 final : public IRenderer
````

Reply:
Done